src/languages/csharp.js

```diff
@@ -91,8 +91,11 @@ export default function(hljs) {
   var GENERIC_MODIFIER = {
     begin: "<",
     end: ">",
-    keywords: "in out",
```

Comment:
Why was this a problem?

Reply:
Because it does not work, I've made it by another way using keywords, and now space is not  including in keyword tags

Reply:
Ah yes, TITLE would eat them...  but the original solution wasn't optimal.  `beginKeywords` is the best quick way to do this!   As you're doing now.